modules/system/models/MailSetting.php

```diff
@@ -85,62 +83,52 @@ public function getSendModeOptions()
             static::MODE_SENDMAIL => 'system::lang.mail.sendmail',
             static::MODE_SMTP     => 'system::lang.mail.smtp',
             static::MODE_MAILGUN  => 'system::lang.mail.mailgun',
-            static::MODE_MANDRILL => 'system::lang.mail.mandrill',
             static::MODE_SES      => 'system::lang.mail.ses',
-            static::MODE_SPARKPOST => 'system::lang.mail.sparkpost',
         ];
     }
 
     public static function applyConfigValues()
     {
         $config = App::make('config');
         $settings = self::instance();
-        $config->set('mail.driver', $settings->send_mode);
+        $config->set('mail.default', $settings->send_mode);
         $config->set('mail.from.name', $settings->sender_name);
         $config->set('mail.from.address', $settings->sender_email);
 
         switch ($settings->send_mode) {
             case self::MODE_SMTP:
-                $config->set('mail.host', $settings->smtp_address);
-                $config->set('mail.port', $settings->smtp_port);
+                $config->set('mail.mailers.smtp.host', $settings->smtp_address);
+                $config->set('mail.mailers.smtp.port', $settings->smtp_port);
                 if ($settings->smtp_authorization) {
-                    $config->set('mail.username', $settings->smtp_user);
-                    $config->set('mail.password', $settings->smtp_password);
+                    $config->set('mail.mailers.smtp.username', $settings->smtp_user);
+                    $config->set('mail.mailers.smtp.password', $settings->smtp_password);
                 }
                 else {
-                    $config->set('mail.username', null);
-                    $config->set('mail.password', null);
+                    $config->set('mail.mailers.smtp.username', null);
+                    $config->set('mail.mailers.smtp.password', null);
                 }
                 if ($settings->smtp_encryption) {
-                    $config->set('mail.encryption', $settings->smtp_encryption);
+                    $config->set('mail.mailers.smtp.encryption', $settings->smtp_encryption);
                 }
                 else {
-                    $config->set('mail.encryption', null);
+                    $config->set('mail.mailers.smtp.encryption', null);
                 }
                 break;
 
             case self::MODE_SENDMAIL:
-                $config->set('mail.sendmail', $settings->sendmail_path);
+                $config->set('mail.mailers.sendmail.path', $settings->sendmail_path);
                 break;
 
             case self::MODE_MAILGUN:
                 $config->set('services.mailgun.domain', $settings->mailgun_domain);
                 $config->set('services.mailgun.secret', $settings->mailgun_secret);
                 break;
```

Comment:
If I remember correctly, the mailgun driver also isn't supported by default in Laravel 9 and will require a driver plugin for as well.

Reply:
I see this in Laravel MailManager:

```
    /**
     * Create an instance of the Symfony Mailgun Transport driver.
     *
     * @param  array  $config
     * @return \Symfony\Component\Mailer\Bridge\Mailgun\Transport\MailgunApiTransport
     */
    protected function createMailgunTransport(array $config)
    {   
        $factory = new MailgunTransportFactory();

        if (! isset($config['secret'])) {
            $config = $this->app['config']->get('services.mailgun', []);
        }

        return $factory->create(new Dsn(
            'mailgun+'.($config['scheme'] ?? 'https'),
            $config['endpoint'] ?? 'default',
            $config['secret'],
            $config['domain']
        ));
    }
```